wgsl/index.bs

```diff
@@ -9346,6 +9354,51 @@ struct __modf_result_vecN {
         [=Component-wise=] when |T| is a vector.
         (SPIR-V OpBitCount)
 
+  <tr algorithm="count trailing zeroes">
+    <td>|T| is [INTEGRAL]
+    <td class="nowrap">`countTrailingZeros(`|e|`:` |T| `) ->` |T|
+    <td>The number of 0 bits in the least significant bit positions
+        of |e|, when |T| is a scalar type.<br>
+        [=Component-wise=] when |T| is a vector.<br>
+        Also known as "ctz" in some languages.<br>
+
+  <tr algorithm="signed find most significant one bit">
+    <td>|T| is [SIGNEDINTEGRAL]
+    <td class="nowrap">`firstBitHigh(`|e|`:` |T| `) ->` |T|
+    <td>For scalar |T|, the result is:
+        <ul>
+        <li>|T|(-1) if |e| is zero, or -1.
+        <li>Otherwise the position of the most signficant bit in
+            |e| that is different from |e|'s sign bit.
+        </ul>
+            
+        Note: Since signed integers use twos-complement representation,
+        the sign bit appears in the most significant bit position.
+
+        [=Component-wise=] when |T| is a vector.<br>
+
+  <tr algorithm="unsigned find most significant one bit">
+    <td>|T| is [UNSIGNEDINTEGRAL]
+    <td class="nowrap">`firstBitHigh(`|e|`:` |T| `) ->` |T|
+    <td>For scalar |T|, the result is:
+        <ul>
+        <li>|T|(-1) if |e| is zero.
+        <li>Otherwise the position of the most signficant 1
+            bit in |e|.
+        </ul>
+        [=Component-wise=] when |T| is a vector.<br>
+
+  <tr algorithm="find least significant one bit">
+    <td>|T| is [INTEGRAL]
+    <td class="nowrap">`firstBitLow(`|e|`:` |T| `) ->` |T|
```

Comment:
This name strikes me as unfortunate.

Reply:
Agreed. First reading I thought this was returning the first found `0`.